.lando.yml

```diff
@@ -21,7 +21,7 @@ services:
         SIMPLETEST_DB: "sqlite://localhost/tmp/db.sqlite"
         BROWSERTEST_OUTPUT_DIRECTORY: '/app/web/sites/simpletest/browser_output'
         BROWSERTEST_OUTPUT_BASE_URL: 'https://drupal-contributions.lndo.site'
-        MINK_DRIVER_ARGS_WEBDRIVER: '["chrome", {"browserName":"chrome","chromeOptions":{"args":["--disable-gpu","--headless", "--no-sandbox"]}}, "http://chrome:9515"]'
+        MINK_DRIVER_ARGS_WEBDRIVER: '["chrome", {"browserName":"chrome","chromeOptions":{"args":["--disable-gpu","--headless"]}}, "http://chromedriver:9515"]'
```

Comment:
Shouldn't `DRUPAL_TEST_WEBDRIVER_HOSTNAME` be changed too?

Reply:
I was wrong.
With the given configuration, the `DRUPAL_TEST_WEBDRIVER_HOSTNAME` value is the right one because the chrome driver container is reachable using the `chrome` hostname. This value is used for nightwatch tests which seem to ignore cookie domain issues.

The real problem is that the `WebDriverTestBase` tests are still not working because they cannot find the chromedriver service at the `http://chromedriver:9515` URL. That's why they appear as `skipped` in the PHPUnit report.

So, if we use `http://chromedriver:9515` the test is skipped because it cannot reach the driver and if we use `http://chrome:9515` which is the appropriate URL with the current lando configuration, the test fails because of the cookie domain error...

When trying to understand what's happening inside the chromedriver service, I can see the following trace (in `/tmp/chromedriver.log`):
```
[1668781158.148][INFO]: [fe198b47912849d98bcf4c66f13b4b93] RESPONSE GetUrl "https://drupal-contributions.lndo.site/"
[1668781158.148][INFO]: [fe198b47912849d98bcf4c66f13b4b93] COMMAND AddCookie {
   "cookie": {
      "domain": "drupal-contributions.lndo.site",
      "expires": 1668861158,
      "name": "SIMPLETEST_USER_AGENT",
      "secure": false,
      "value": "simpletest20588224%3A1668781158%3A63779466232a65.73180091%3ABGYNVRC51qyN8Ch_xyVlXDJwHb1XmKrfcCbgQWiUsa4"
   }
}
```

That helped me to understand that we are requesting **https** but the cookie is set with `"secure": false` so that means that somehow Drupal doesn't understand he is serving secure content.

If we replace `SIMPLETEST_BASE_URL` by an `http` URL, the tests are working again!... but most of them are failing T_T

I had to dig a lot more to understand why the chromedriver was crashing despite the `--no-sandbox` option you added. As usual, the answer came from [stackoverflow](https://stackoverflow.com/a/53970825) and was talking about the `/dev/shm` partition size. The easiest way to fix it seem to be mounting it as a volume linked to the host machine partition.

In the end I finally found a way to run the tests with the following changes **only**:
1. changing both `SIMPLETEST_BASE_URL` and `BROWSERTEST_OUTPUT_BASE_URL` to *http*
```
        SIMPLETEST_BASE_URL: "http://drupal-contributions.lndo.site/"
...
        BROWSERTEST_OUTPUT_BASE_URL: 'http://drupal-contributions.lndo.site'
```
3. adding a volume on the `chrome` service:
```
  chrome:
    type: compose
    app_mount: false
    services:
      image: drupalci/webdriver-chromedriver:production
      volumes:
        - /dev/shm:/dev/shm
      command: chromedriver --log-path=/tmp/chromedriver.log --allowed-origins=* --verbose --whitelisted-ips=
```

Now, all tests are working \o/

Reply:
all the `chromedriver` hosts are because that's what I named my service but drupal-contributions uses `chrome` for the service so they should be converted.

As far as the sandbox stuff, that's interesting. I wonder if its environment specific because I was getting similar errors with `--no-sandbox` being needed but not solving the whole shared memory thing but the SO search ended with me adding the `seccomp:unconfined` line which fixed it on my machine. I guess you may need one of the other depending on system configuration but I can't see a problem with adding both. Well, other then the fact both are turning off container protections but I think that's going to be the price of doing business with chrome.